lib/usd/translators/scopeReader.cpp

```diff
@@ -27,7 +29,23 @@ PXR_NAMESPACE_OPEN_SCOPE
 PXRUSDMAYA_DEFINE_READER(UsdGeomScope, args, context)
 {
     const UsdPrim& usdPrim = args.GetUsdPrim();
-    MObject        parentNode = context->GetMayaNode(usdPrim.GetPath().GetParentPath(), true);
+
+    // If this is a Looks/Materials scope that contains only UsdShade nodes, just skip.
+    bool hasLookData = false;
+    bool hasNonLookData = false;
+    for (auto const& child : usdPrim.GetChildren()) {
+        if (child.IsA<UsdShadeNodeGraph>() || child.IsA<UsdShadeShader>()) {
+            hasLookData = true;
+        } else {
+            hasNonLookData = true;
+            break;
+        }
+    }
+    if (hasLookData && !hasNonLookData) {
+        return false;
+    }
```

Comment:
Makes sense to me, but a few somewhat cosmetic notes:
- Though `Looks` is unfortunately still the default for the material scope name, I think we'd like to transition away from using that term, and really we're just interested in finding *any* `UsdGeomScope` prims with authored scene description below them that is only shading information.
- It might be a bit more robust to check whether the child prim is compatible with `UsdShadeConnectableAPI` rather than looking specifically at its node graph-ness or shader-ness.

So with those two small tweaks, something like this?:
```C++
    // If this scope contains only UsdShade nodes, just skip.
    bool hasShadingData = false;
    bool hasNonShadingData = false;
    for (const auto& child : usdPrim.GetChildren()) {
        if (UsdShadeConnectableAPI(child)) {
            hasShadingData = true;
        } else {
            hasNonShadingData = true;
            break;
        }
    }
    if (hasShadingData && !hasNonShadingData) {
        return false;
    }
```